scripts/appleseedMaya/renderGlobals.py

```diff
@@ -355,6 +355,10 @@ def __adaptiveSamplerChanged(self, value):
         self._uis["minPixelSamples"].setEnable(value)
         self._uis["batchSampleSize"].setEnable(value)
         self._uis["sampleNoiseThreshold"].setEnable(value)
+        if value:
+            mc.setAttr("appleseedRenderGlobals.samples", 256)
+        else:
+            mc.setAttr("appleseedRenderGlobals.samples", 32)
```

Comment:
In appleseed-max the default value is 64 spp:

![image](https://user-images.githubusercontent.com/321290/69008686-9ed34880-094d-11ea-9741-4612d6e3b81b.png)

Reply:
Yes, we can set it here to 64 as well. The reason why I had 32 spp in Maya is that it gives a quick scene overview. In Max we have active shade mode with progressive rendering for this, but in Maya we don't have an analogue ipr mode.

Let me know what you think is better, having it 64 spp like in studio and Max or lower like 32 (in blenderseed it is 16 spp default).